api/agentpb/agent.proto

```diff
@@ -91,8 +98,10 @@ message StartActionRequest {
   }
   // MySQLExplainParams describes MySQL Explain action parameters.
   message MySQLExplainParams {
-    // TODO
-
+    string dsn = 1;
+    string db = 2;
```

Comment:
Isn't database name a part of DSN?

Reply:
Yes, but I'm not sure it enough:

https://github.com/percona/qan-agent/blob/9399e1fe46df02e988838816c0578282918b0b8c/query/plugin/mysql/explain/explain.go#L62-L69

Reply:
We will extract it from DSN.
If we have two options to pass database name, we will have 4 combinations we will need to handle. I don't think we should.